core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.ts

```diff
@@ -90,21 +88,23 @@ export class ContributorAdminStatsTable implements OnInit {
   itemsPerPageChoice: number[] = [20, 50, 100];
   itemsPerPage: number = 20;
   statsPageNumber: number = 0;
+  maximumExpectedItems: number = 1000;
```

Comment:
Since this state never changes, consider making this a constant.

Reply:
Done